pkg/urbit/daemon/main.c

```diff
@@ -164,10 +163,6 @@ _main_getopt(c3_i argc, c3_c** argv)
         u3_Host.ops_u.key_c = strdup(optarg);
         break;
       }
-      case 'm': {
-        u3_Host.ops_u.sap_c = strdup(optarg);
```

Comment:
Please also remove `sap_c` and `etn` from `u3_opts`. (I know that struct is already out of date, but I don't want it to get worse ...)

Reply:
Done